src/App.js

```diff
@@ -29,26 +30,24 @@ export default function App() {
   return (
     <Router>
       <Routes>
-        <Route exact path="/onboarding-1" element={<Onboarding1 />} />
-        <Route exact path="/onboarding-2" element={<Onboarding2 />} />
-        <Route exact path="/onboarding-3" element={<Onboarding3 />} />
-        <Route exact path="/onboarding-4" element={<Onboarding4 />} />
-        <Route exact path="/onboarding-5" element={<Onboarding5 />} />
-        <Route path="/" element={<Frame />}>
-          <Route path="/dashboard" element={<Dashboard />} />
-          <Route path="/rsvp-preview" element={<RSVP />} />
-          <Route path="/invite" element={<InvitationForm />} />
-          <Route path="/registry" element={<Registry />} />
-          <Route path="/registrystart" element={<RegistryStart />} />
-          <Route path="/registryadd" element={<RegistryAdd />} />
-          <Route path="/guest-invite" element={<Invite />} />
-          <Route path="/rsvp-form" element={<RsvpForm />} />
-          <Route path="/browse-theme" element={<BrowseTheme />} />
-          <Route path="/choose-content" element={<ChooseContent />} />
-          <Route path="/invitation-form" element={<InvitationForm />} />
-          <Route path="/rsvp-preview" element={<RSVP />} />
-          <Route path="/guestlist" element={<GuestListMainPage />} />
-        </Route>
+        <Route
+          path="/"
+          element={
+            <Onboarding1 />
+            // <h2>Just localhost</h2>
+          }
+        />
+        <Route path="/registry" element={<Registry />} />
+        <Route path="/registrystart" element={<RegistryStart />} />
+        <Route path="/registryadd" element={<RegistryAdd />} />
+        <Route path="/guestregistry" element={<GuestRegistry />} />
+        <Route path="/guest-invite" element={<Invite />} />
+        <Route path="/rsvp-form" element={<RsvpForm />} />
+        <Route path="/browse-theme" element={<BrowseTheme />} />
+        <Route path="/choose-content" element={<ChooseContent />} />
+        <Route path="/invitation-form" element={<InvitationForm />} />
+        <Route path="/rsvp-preview" element={<RSVP />} />
+        <Route path="/guestlist" element={<GuestListMainPage />} />
```

Comment:
I think you may have accepted the wrong changes when pulling, these are the old changes before the nav/footer were added to the route